src/definitions/modules/toast.less

```diff
@@ -119,7 +158,34 @@
     font-size: @toastIconFontSize;
     opacity: @toastIconOpacity;
 }
+
+.icon.flat.toast > .icon:not(.close) {
+  opacity:1;
+}
+
 .icon.toast > .content {
     display: inline-block;
     vertical-align: @toastContentVerticalAlign;
-}
\ No newline at end of file
+}
+
+/*---------------
+   Progress Bar
+ ----------------*/
+
+.toast-box .attached.progress .bar {
```

Comment:
This toast box should host all toasts, not just the toasts that have progress bar.

Reply:
toast-box is only used when a progress bar is enabled. otherwise the additional wrapper toast-box is  not needed as before and the additional div is not rendered

Reply:
This (simplified) is still used when a progressbar should not be enabled
```html
<div class="toast-container">
	<div class="ui toast></div>
</div>
```


This (simplified) is used when a progressbar is enabled (to wrap the progressbar correctly underneath the toast component
```html
<div class="toast-container">
	<div class="ui toast-box">
		<div class="ui toast></div>
		<div class="bottom attached progress">
			<div class="bar"></div>
		</div>
	</div>
</div>
```

Reply:
Agreed but adding a parent to only some kind of toast might break some behaviors in the long term. For now the toast component is young and pretty simple, but with time its complexity will increase and we must think about it now

Reply:
I agree with @prudho. Maybe we should do something like this
```html
<div class="toast-container">
  <div class="ui toast">
    <div class="content">
      <!-- title, message etc. -->
    </div>
    <div class="bottom attached progress">
      <div class="bar"></div>
    </div>
  </div>
</div>
```

Reply:
Basically good idea. But how about supporting the message component easily then without the need to somehow recreate everything from message into toast aswell?
I was very glad, the complete message component functionality was working with toast just by changing the className.
At first, i was thinking of getting rid of the "own" toast class in favour of simply use the message component completely. But, again, i did not want to break everything @prudho already did :smile: 
For now i would just change to code to render "toast-box" wrapping all the time instead of just render it when a progressbar should be shown. What do you think?

Reply:
By commit ccf7ad077c0cd1fd5e178acec49ac7b04b940eaf the parent element "text-box" is now always rendered. 

Btw: Also supported "top attached progress" and fixed icon display with this commit :grin: